docker/image/git_add_files.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+import argparse
+import git
+import re
+
+parser = argparse.ArgumentParser(
```

Comment:
@apojomovsky consider moving this below an `if __name__ == '__main__': ` clause.

Reply:
done, thanks!